recipes/corrade/all/conanfile.py

```diff
@@ -1,21 +1,25 @@
-from conan.tools.microsoft import is_msvc
-from conan.tools.microsoft.visual import vs_ide_version
-from conans import ConanFile, CMake, tools
-from conans.errors import ConanInvalidConfiguration
-import functools
 import os
 
-required_conan_version = ">=1.45.0"
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import cross_building
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rmdir
+from conan.tools.microsoft import is_msvc
+from conan.tools.microsoft.visual import vs_ide_version, check_min_vs
```

Comment:
I am allowing this because it was already broken, but ``vs_ide_version`` is at the moment a private function, and can't be used, it can break anytime.